beamer/tests/conftest.py

```diff
@@ -2,57 +2,54 @@
 import os
 from dataclasses import dataclass
 
-import brownie
+import ape
 import eth_account
 import pytest
-from brownie import (
-    FillManager,
-    RequestManager,
-    Resolver,
-    TestL1Messenger,
-    TestL2Messenger,
-    accounts,
-)
+from ape import chain, project
```

Comment:
Nitpick: I would have preferred to use full qualification for `chain` (`ape.chain`) to make it more explicit and avoid possible variable shadowing.